python/docs/source/getting_started/index.rst

```diff
@@ -40,3 +40,4 @@ The list below is the contents of this quickstart page:
    quickstart_df
    quickstart_connect
    quickstart_ps
+   testing_pyspark
```

Comment:
Do you plan to create a separate binder link? or just keep this as a document without binder link?

Reply:
planning to keep as a document, with no binder link